diff --git a/harness/sta.js b/harness/sta.js
index ac59d3a110..826396185f 100644
--- a/harness/sta.js
+++ b/harness/sta.js
@@ -18,8 +18,8 @@ Test262Error.prototype.toString = function () {
   return "Test262Error: " + this.message;
 };
 
-Test262Error.thrower = (...args) => {
-  throw new Test262Error(...args);
+Test262Error.thrower = function(msg) {
+  throw new Test262Error(msg);
 };
 
 var $ERROR = Test262Error.thrower;
